docs/docs/guides/core-types/evaluation_playground.md

```diff
@@ -0,0 +1,109 @@
+
+For example, you can open the Evaluation Playground, add two models you've previously saved to W&B, and then evaluate their performance based on a new or previously saved question-answer style evaluation dataset. You can then create a new model in the interface, add system prompts to it, and then run a new evaluation on all three models to see how they perform against one another.
+
+![Evaluation Playground Interface](../evaluation/img/eval-playground-ui.png)
+
+## Set up an evaluation in the playground
+
+To set up an evaluation in the Evaluation Playground:
+
+1. Open the W&B UI, and then open the project that you want to perform the evaluation in. This opens the Traces page. 
+2. From the Traces page, click the **Playground** icon in the left-menu, and then select the **Evaluate** tab on the Playground page. On the Evaluate page, you can either: 
+    * **Load a demo example**: This loads a predefined configuration that evaluates MoonshotAI Kimi K2 model on expected output and uses an LLM judge to determine its correctness. You can use this configuration to experiment with the interface.
+    * **Start from scratch**: This loads a blank configuration for you to build upon.
+3. If you selected **Start from scratch**, add a descriptive title and description for your evaluation in the **Title** and **Description** fields.
+
+Follow the instructions in the next sections to set up your datasets, models, and scorers.
+
+### Add a dataset
+
+[Datasets](../core-types/datasets) are organized collections example user inputs and your expected responses from models. During evaluation, the playground feeds each test input into your model, collects its output, and then scores the output based on a metric you've selected, such as correctness. You can choose to create a dataset in the UI, add an existing dataset already saved to your project, or upload a new dataset.
+
+You can upload datasets in the following formats:
+
+* `.csv`
+* `.tsv` 
+* `.json`
+* `.jsonl`
+
+See the Datasets pages for more information on how to format and save datasets to Weave.
+
+To add a dataset in the **Dataset** section: 
+
+1. Click the dropdown menu and then select either: 
+    * **Start from scratch** to create a new dataset in the UI.
+    * **Upload a file** to upload a dataset from your local machine.
+    * An existing dataset already saved to your project.
+2. Optional: Click **Save** to save the dataset to your project for later use.
+
+Once you've selected an option, the dataset is visible in the right pane of the UI and you can edit each field as necessary by clicking on it. You can also add new rows to the dataset by clicking **Add row**.
+
+### Add a model
+
+[Models](https://docs.wandb.ai/guides/models/) are trained systems that generate output based on user input. You can select existing models in your project to create new ones to evaluate.
```

Comment:
"Models" in the weave world are very rarely trained. we should not be linking to wandb models here. We should be linking to Weave models. they are distinct things with distinct properties.

Note: you cannot just choose any model for these evaluations - they must be models created in the playground interface (the user likely has many models which will not be available here)